### ISSUE TITLE
Suggestion: find_labels_adjacency() function for cortical labels

### ISSUE THREAD

**Issue by anntabueva** [mne-tools/mne-python#8646]
#### Describe the new feature or enhancement
Apparently, among functions that compute adjacency matrix for channels and sources, MNE currently does not have a function that computes the spatial adjacency matrix for cortical labels (e.g., Desikan-Killiani atlas with 68 cortical labels). Similar to mne.channels.find_ch_adjacency() function, such function should have list of labels as input and return the scipy.sparse.csr_matrix of labels adjacency with shape (n_labels, n_labels). The output of such function could be used for cluster permutational statistics.

#### Describe your proposed implementation
I suggest it can be done relatively easy using FreeSurfer triangulations.

**Comment by lqxxgnd** [comment]
i agree with you!

**Comment by larsoner** [comment]
Agreed this would be good, if someone is up for giving it a shot feel free to open a PR!

**Comment by wmvanvliet** [comment]
Here is a gist which an implementation: https://gist.github.com/wmvanvliet/fc0f1a3334955698303841fcc8a8d85d

**Comment by wmvanvliet** [comment]
@raphbrd, @payamsash

**Comment by wmvanvliet** [comment]
If we want to support volumetric labels,we may first have to create a `VolLabel` class, because right now all the functions dealing with volumetric labels take an `aseg`/`labels` pair of arguments.

**Comment by larsoner** [comment]
Yeah for the first PR I would just support handling a cortical parcellation and/or list of cortical surface labels. Volume could come later
